{{cookiecutter.project_slug}}/backend/pyproject.toml

```
@@ -1,19 +1,11 @@
-[[source]]
-name = "pypi"
-url = "https://pypi.org/simple"
-verify_ssl = true
+[tool.poetry]
+name = "{{cookiecutter.project_slug}}"
+version = "{{cookiecutter.project_version}}"
```

```suggestion
name = "backend"
version = "0.1.0"
```

Let's have this file without template variables so that it can work as is. This might also help bots reading it, e.g. Dependabot.